modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1663,13 +1663,13 @@ public FeatureTypeStyle transformInternal(MBStyle styleContext) {
         Fill fill = sf.fill(null, textColor(), textOpacity());
 
 
-        //Font not supported until glyphs are working
-        Font font = sb.createFont(ff.literal("" /* transformer.getDefaultFonts() */), ff.literal("normal"),
-                ff.literal("normal"), textSize());
-        /* else {
-            font = sb.createFont(ff.literal(getTextFont()), ff.literal("normal"),
-                    ff.literal("normal"), textSize());
-        } */
+        Font font = sb.createFont(ff.literal(""), ff.literal("normal"), ff.literal("normal"), textSize());
+        if (getTextFont() != null) {
```

Comment:
I may be mistaken, but this will probably die if fonts are specified as a function in the style. It may be a little premature to support function-defined fonts at this point -- just something to be aware of.